src/GitTfs/Core/GitTfsRemote.cs

```diff
@@ -702,8 +702,22 @@ public void QuickFetch()
 
         public void QuickFetch(int changesetId)
         {
-            var changeset = Tfs.GetChangeset(changesetId, this);
-            quickFetch(changeset);
+            try
+            {
+                var changeset = Tfs.GetChangeset(changesetId, this);
+                quickFetch(changeset);
+            }
+            catch (Exception ex)
+            {
+                // Ignore exceptions "TF14098: Access Denied: User ??? needs
+                // Read permission(s) for at least one item in changeset ???."
+                if (!ex.Message.Contains("TF14098"))
+                {
+                    throw;
+                }
+
+                Trace.WriteLine("Quick fetch failed: " + ex.Message);
```

Comment:
In this case, would there end up being any contents in the repo? If not, maybe it doesn't make sense to `catch` for `quick-clone`?

Reply:
I agree that no contents would be in the repo in case of `quick-clone`. However, it looks like as if `QuickFetch` is also called within a full fetch for the very first change set:
https://github.com/git-tfs/git-tfs/blob/c89e8ddc438a478401738c42ad835d6fda68499b/src/GitTfs/Commands/Fetch.cs#L192

Do you see an easy way to distinguish the two cases?

Reply:
Ah, right, I forgot about that. It'd still be good to make `quick-clone` fail, but not at the expense of breaking full clones or fetches. So, if it's not too cumbersome to separate the behavior of the two cases, please do.